docs/core/compatibility/code-analysis/5.0/ca2014-stackalloc-in-loops.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
This is not actually a new file, just a rename of an include and adjustment of the H1-H4 levels and relative links.